templates/feature_group/sdk_delete_pre_build_request.go.tpl

```diff
@@ -0,0 +1,4 @@
+// If a delete failed, requeue on delete.
```

Comment:
why do we need pre_build_request and post_request ?
isn't `sdk_delete_post_request.go.tpl ` enough ?

Reply:
Good suggestion! I looked more into it, as I had modeled this after a similar elastic-cache PR for requeue on "deleting" state, and concluded that for a delete failed state (rather than deleting) this pre_build_request should not be used. I went ahead and removed the pre_build_request hook and code. (We can eventually check this further in unit testing)